server/src/main/java/org/elasticsearch/action/admin/indices/rollover/RolloverRequestBuilder.java

```diff
@@ -58,6 +58,11 @@ public RolloverRequestBuilder addMaxIndexSizeCondition(ByteSizeValue size){
         return this;
     }
 
+    public RolloverRequestBuilder addMaxIndexSinglePrimarySizeCondition(ByteSizeValue size){
```

Comment:
Super minor nit
```suggestion
    public RolloverRequestBuilder addMaxIndexSinglePrimarySizeCondition(ByteSizeValue size) {
```

(and same comment about dropping `Index`)

Reply:
Fixed the whitespace in af3f3e9 -- still holding off on the method rename for now.